src/main/java/com/stripe/model/SubscriptionSchedule.java

```diff
@@ -131,7 +129,7 @@
 
   /** Configuration for the subscription schedule's phases. */
   @SerializedName("phases")
-  List<Phase> phases;
+  List<SubscriptionSchedule.Phase> phases;
```

Comment:
What is this diff?

Reply:
This seems safe: it looks like the prettier generator is using `OuterClass.InnerClass` instead of just `InnerClass`.